Domain/ProductManagement/Product.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace LenaInventoryManagementSystem.Domain.ProductManagement
```

Comment:
NIT: you can just do `﻿namespace LenaInventoryManagementSystem.Domain.ProductManagement;` which will remove the need of adding { } to wrap the rest of the code